packages/engine/lib/execution/src/package/simulation/context/agent_messages/package.js

```diff
@@ -1,4 +1,4 @@
-import * as hash_util from "./lib/execution/src/runner/javascript/hash_util.js";
+import * as hash_util from "./src/runner/javascript/hash_util";
```

Comment:
Imports have to be relative to the crate root now and not the workspace.
Before they work because we usually `cargo` from the workspace root but it was not super robust.